### PR TITLE
Add a CONTRIBUTORS.MD file to credit people who may not appear in the commit logs

### PR DESCRIPTION
**Pull request by jamescarlyle**
Added a single text file containing a full list of contributors to Corda code and Corda thinking via the R3 Architecture Working Group, including all comments to Wiki pages. I will revisit this and add to the list those bank developers who have contributed during on-site visits to R3.

# PR Checklist:

- [x] Have you run the unit, integration and smoke tests as described here? https://docs.corda.net/head/testing.html
- [x] If you added/changed public APIs, did you write/update the JavaDocs?
- [x] If the changes are of interest to application developers, have you added them to the changelog, and potentially release notes?
- [x] If you are contributing for the first time, please read the agreement in CONTRIBUTING.md now and add to this Pull Request that you agree to it.

Thanks for your code, it's appreciated! :)

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+* Daniel Roig (SEB)
+* David Lee (BCS)
+* Farzad Pezeshkpour (RBS)
+* Frederic Dalibard (Natixis)
+* Garrett Macey (Wells Fargo)
+* Gavin Thomas (R3)
+* George Marcel Smetana (Bradesco)
+* Giulio Katis (Westpac)
+* Giuseppe Cardone (Intesa Sanpaolo)
+* Guy Hochstetler (IBM)
```
Comment (gendal) [inline CONTRIBUTORS.md]: IBM?

**Comment by gendal** [comment]
other names off the top of my head:

Maka
Rodrigo
Ian Grigg
anybody from legal/smart contract template work?

**Comment by mikehearn** [comment]
Can you provide a PR description please?

**Comment by mikehearn** [comment]
Nice! Maybe we should split this into R3 and external sections?

**Comment by jamescarlyle** [comment]
I thought of doing that, but wanted to emphasise that this is truly a collaborative effort, and therefore I deliberately mixed all contributors together regardless of employer.

File: CONTRIBUTORS.md (hunk 1 above)
Comment (jamescarlyle) [reply CONTRIBUTORS.md]: His first contribution was from that period. But I will add an overall note saying that some people may have changed organisation.

Comment (jamescarlyle) [reply CONTRIBUTORS.md]: Have added other contributors in response to Gendal

**Pull request merged**